db/migrations/20240410141239-add-constraints-on-remediation-name.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+    up: async (q, {STRING}) => {
+        const transaction = await q.sequelize.transaction()
+        try {
+            // Add unique constraint to remediations name column based on tenant_org_id
+            await q.addConstraint('remediations', {
+                name: 'name_and_tenant_org_id',
+                type: 'UNIQUE',
+                fields: ['name', 'tenant_org_id']
+            }, { transaction });
+
+            // Add NOT NULL constraint to name
+            // Fixing duplicates should make sure there are no NULL name values because either
+            await q.changeColumn('remediations', 'name', {type: STRING, allowNull: false}, { transaction });
```

Comment:
Unless I missed it we never changed the existing records with null names...

Reply:
I was for some reason thinking that this would be resolved by updating the duplicates but I guess there might be a null name that isn't a duplicate within an org 🤔 I can add a query that updates remaining null values! I'm guessing there won't be a lot so it probably wouldn't add too much time.